src/tests/useQuery.test.js

```diff
@@ -152,9 +147,9 @@ describe('useQuery', () => {
       )
     }
 
-    const rendered = render(<Page />)
+    const { findByText } = render(<Page />)
```

Comment:
Suspicion: due to GC happening on subsequent tests that was fixed in #332, this may have been passing as a fluke. The test before this ensures it's in a `loading` state by default, which is what the rendered output looks like for this test when it was failing.